.circleci/config.yml

```diff
@@ -5,6 +5,7 @@ my-steps: &steps
   - run: sudo pip install -r requirements.txt
   - run: sudo pip install flake8 pytest
   - run: python --version ; pip --version ; pwd ; ls -l
+  - run: pip install --user .
```

Comment:
Not needed. CircleCI (like Travis CI) does this automatically.

Reply:
Hmm .. I must admit, I do not know much about Circle .. but it failed before I added this.

Have a look here please: https://circleci.com/gh/ggtracker/sc2reader/259
```
Traceback:
test_replays/test_replays.py:21: in <module>
    import sc2reader
E   ModuleNotFoundError: No module named 'sc2reader'
```

Reply:
OK.  Go with what works.

Reply:
ok